REVIEW_COPY.md

```diff
@@ -0,0 +1,312 @@
+# Drag & Drop without Draggables & Dropzones
+
+Drag & drop UIs have been around for a while, and so you would be forgiven for thinking they're a solved problem. Certainly there's high quality libraries to help us build drag & drop UIs, and these days we even have an official HTML5 drag & drop API! Why then is building drag & drop UIs so hard?
+
+Having built some drag & drop UIs I've come to the conclusion a big part of the problem is our choice of tools. Those tools are the draggables and dropzones most libraries provide for building drag & drop interactions. Not using draggables and dropzones to build UIs often simplifies their construction massively.
+
+That's a pretty sweeping statement for a broad term like 'drag & drop', so I'd better back it up! To do so I'll show you three examples of drag & drop UIs built without draggables or dropzones. These examples come from real projects, but I've stripped them down to their drag & drop essentials for this post. I'll include what I believe to be the most relevant code snippets in those post, and invite you to look at [the full source code] if you'd like to learn more!
+
+Before we start with those examples, lets start with a quick refresher on draggables and dropzones.
+
+## A Refresher on Draggables & Dropzones
+
+To learn what draggables and dropzones are lets look at [Trello], an application where they are easy to identify. In Trello you can drag cards between lists. This makes cards draggables and lists dropzones.
```

Comment:
💭 be careful about saying something is "easy" or "obvious". The audience may have folks who are newer to programming, and to whom these things may not be so easy. I think you could just get rid of this part of the sentence, though, since you identify the draggables and dropzones immediately following.